hack/package-hasher/Containerfile.apk.hasher

```diff
@@ -19,6 +19,9 @@ RUN repro-get hash generate > SHA256SUMS
 # Export the apks
 RUN repro-get cache export repository-apk
 
+# Sort the hashes
+RUN sort -k2 -o SHA256SUMS SHA256SUMS
```

Comment:
I'd suggest to also sort the current file, so that we can already enjoy the sorting for the next change.